app/views/documents/show/_actions.html.erb

```diff
@@ -84,7 +84,17 @@
 
     <% if @edition.document.live_edition %>
       <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
-      <%= link_to "View published edition on GOV.UK", EditionUrl.new(@edition.document.live_edition).public_url, class: "govuk-link govuk-link--no-visited-state" %>
+      <ul class="govuk-list">
```

Comment:
We can add a `govuk-!-margin-0` class to this list if we want to remove the bottom margin.

Reply:
Done 👍